tests/test_shell.py

```diff
@@ -255,15 +255,15 @@ def test_find(self):
                 self.call(
                     "python -m signac find ".split() + ['{"doc.a": ' + str(i) + "}"]
                 ).strip()
-                == [job.id for job in project.find_jobs(doc_filter={"a": i})][0]
+                == [job.id for job in project.find_jobs({"doc.a": i})][0]
             )
 
         for i in range(1, 4):
             assert (
                 self.call(
                     "python -m signac find ".split() + ['{"doc.b": ' + str(i) + "}"]
                 ).strip()
-                == [job.id for job in project.find_jobs(doc_filter={"b": i})][0]
+                == [job.id for job in project.find_jobs({"doc.b": i})][0]
```

Comment:
Please duplicate the test instead of altering it to ensure that we do not introduce a regression in future releases.

Reply:
This case is primarily a test of the CLI. The call to `find_jobs` is only used as a comparison value here, and it's not intending to test `find_jobs`. I think we achieve sufficient coverage of `doc_filter` parameters through the extensive testing of `find_jobs`. If you still think test duplication is warranted, then I will do so.

Reply:
You are correct and I am sorry that I didn't see this comment. All good.